waspc/examples/todo-typescript/src/ChatPage.tsx

```diff
@@ -45,7 +45,7 @@ export const ChatPage = () => {
 
   return (
     <>
-      <h2>Chat</h2>
+      <h2 className="text-2xl font-bold mb-4">Chat</h2>
```

Comment:
What happened here :D?

Reply:
We need to have some change in `waspc` folder to trigger the CI